packages/cli-kit/src/git.ts

```diff
@@ -12,6 +12,10 @@ export const GitNotPresentError = () => {
   )
 }
 
+export const OutsideGitDirectoryError = () => {
```

Comment:
I'd include in the error the directory tied to this error. @MeredithCastile what do you think about the error message?